### PR TITLE
[v2.4-branch] manifest: nrf_modem v2.4.1

### PR DESCRIPTION
**Pull request by eivindj-nordic**
nrf_modem v2.4.1 release.

See CHANGELOG file in nrfxlib for details.

### CODE REVIEW

**Comment by lemrey** [comment]
Do we need a changelog entry for this, @eivindj-nordic ?

**Comment by eivindj-nordic** [comment]
> Do we need a changelog entry for this, @eivindj-nordic ?

I'm not sure, it is in the nrfxlib changelog.

**Comment by tony-le-24** [comment]
There is an issue with back-end in FOTA tests of the test_fw-nrfconnect-nrf-iot_cloud plan recently. Our cloud BE sometimes didn't send the message to device to let it know about FOTA jobs. It should be fixed in the next release of the nRF prod cloud. I disabled those tests in master branch already (not this v2.4-branch). This PR looks good to me.

**Comment by eivindj-nordic** [comment]
`CI/Jenkins/test-fw-nrfconnect-nrf-iot_lwm2m` and `CI/Jenkins/test-fw-nrfconnect-nrf-iot_zephyr_lwm2m` were passing on a previous CI run, without changes to the PR since then.

**Comment by jvaliharju** [comment]
Due to issues in test execution, the test results regarding modem-shell and positioning are reported as failed, but are in fact all green when manually executed. Ok to merge PR.

**Review by lemrey — APPROVED** [review]

**Review by MirkoCovizzi — APPROVED** [review]

**Review by nordic-hani — APPROVED** [review]

File: west.yml [hunk 1]
```diff
@@ -131,7 +131,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: v2.4.1
+      revision: pull/1057/head
```
Comment (rlubos) [inline west.yml:134]: Please update SHA: `30841c2ad6d3b022406b167ab0ebaf46e8901c56`

**Review by rlubos — APPROVED** [review]

**Pull request merged**